doc/user_guide/user_guide.md

```diff
@@ -43,7 +43,71 @@ We recommend the following requirements for the VM instance:
 | RAM  | 2GB     | 8GB         |
 | CPU  | 1 Core  | 2 Cores     |
 
-### Password
+### Use the AMI
+
+#### Overview
+
+1. Create a Security Group with open inbound ports for `ssh` (22) and `Jupyter` (8888)
+2. Starter the EC-2 instance
+
+#### Step-by-step
+
+1. Go to [AWS Console](https://aws.amazon.com/console/)
+2. Go to EC2 (in the search field at the top type EC2 and click on the result)
+3. Create a security group for ssh and (optionally) Jupyter:
+    - In the navigation bar on the left select "Security Groups"
+    - Click button "Create Security Group"
+    - Choose name and VPC
+    - For outbound rules keep the default
+    - Create  inbound rules: 
+      - One of type `ssh`
+      - If you plan to connect to the Jupyter lab, add another rule of type "Custom TCP" with port 8888. **Important**: With this rule you expose the Jupyter lab to the internet; anybody who has access to the password will be able to execute commands. For a minimum of security you should change the default Jupyter password. Details about how to do that will be shown in the login screen. But, you should consider to use only `ssh`.
+ 4. Go back to the EC2 console
+ 5. Launch the EC2 instance:
+     - In the navigation bar on the left select "Instances"
+     - Click button "Launch instances"
+     - At field "Application and OS Images" select the AMI id of the developer sandbox (found in the [release notes](https://github.com/exasol/script-languages-developer-sandbox/releases/latest))
+     - Select an appropriate instance type (at least "Small")
```

Comment:
What is Small, this is not an instance type

Reply:
changed